### ISSUE TITLE
Anchors & anchor positions

### ISSUE THREAD

**Issue by frankrolf** [adobe-fonts/source-sans-pro#103]
`acircumflexacute.alt` has anchors, which seems to be un-intentional:
![screen shot 2016-11-12 at 12 12 12](https://cloud.githubusercontent.com/assets/2049645/20240618/d1dcd32a-a8d1-11e6-9a20-bf9c89206bf1.png)

The bottom anchor in `gammalatin` needs re-positioning:
![screen shot 2016-11-12 at 12 13 02](https://cloud.githubusercontent.com/assets/2049645/20240619/e41dad0c-a8d1-11e6-9f21-c9f6512d157a.png)

The same anchor issue also exists in `thorn`, `hturned`, `rhookturned`, `rtail`, and (on the top) in `yturned`.

**Comment by frankrolf** [comment]
Anchors in `Chi.sc` are horizontally shifted:
![screen shot 2016-11-12 at 12 30 03](https://cloud.githubusercontent.com/assets/2049645/20240698/d30766c8-a8d3-11e6-9855-2f7d03feabf5.png)

Anchors in `Zhe.sc` are completely off.
![screen shot 2016-11-12 at 12 30 23](https://cloud.githubusercontent.com/assets/2049645/20240702/e9619a74-a8d3-11e6-8ae1-f92c0a78c356.png)

**Comment by frankrolf** [comment]
More observations about the consistency of anchors:
```
Ohorn has anchors aboveUC, belowLC.
ohorn has anchors aboveLC, belowLC.
Ohorn.sc has anchors aboveUC, belowLC, ogonekUC.

Tbar has no anchors.
tbar has no anchors.
Tbar.sc has anchors aboveUC, belowLC, baseLC.

# Alpha has no anchors.
# alpha has anchors aboveLC, belowLC.
# Alpha.sc has anchors aboveLC, belowLC.
# (this is probably fine)

Beta has no anchors.
beta has no anchors.
Beta.sc has anchors aboveLC, belowLC.

# Epsilon has no anchors.
# epsilon has anchors aboveLC, belowLC.
# Epsilon.sc has anchors aboveLC, belowLC.
# (this is probably fine)

Zeta has no anchors.
zeta has no anchors.
Zeta.sc has anchors aboveLC, belowLC.

# Eta has no anchors.
# eta has anchors aboveLC, belowLC.
# Eta.sc has anchors aboveLC, belowLC.
# (this is probably fine)

Iota has no anchors.
iota has one anchor aboveLC.
Iota.sc has anchors aboveLC, belowLC.

Kappa has no anchors.
kappa has no anchors.
Kappa.sc has anchors aboveLC, belowLC.

Mu has no anchors.
mu has no anchors.
Mu.sc has anchors aboveLC, belowLC.

Nu has no anchors.
nu has no anchors.
Nu.sc has anchors aboveLC, belowLC.

Omicron has no anchors.
omicron has one anchor aboveLC.
Omicron.sc has anchors aboveLC, belowLC.

Rho has no anchors.
rho has one anchor aboveLC.
Rho.sc has anchors aboveLC, belowLC.

Tau has no anchors.
tau has no anchors.
Tau.sc has anchors aboveLC, belowLC, baseLC.

Upsilon has no anchors.
upsilon has one anchor aboveLC.
Upsilon.sc has anchors aboveLC, belowLC.

Chi has no anchors.
chi has no anchors.
Chi.sc has anchors aboveUC, belowLC, baseLC.

Ie has one anchor aboveUC.
ie has one anchor aboveLC.
Ie.sc has anchors aboveUC, belowLC.

```

**Comment by erniemarch** [comment]
Checked in build 2.036 roman and 1.086 italic. In the Roman, the gammalatin looks fixed, the chi.sc looks fixed, the Zhe.sc is still off to the right (see attached) and the wsuperior seems to have the attachment point near the baseline rather than under the glyph (see attached).
![zhe_sc_mark](https://user-images.githubusercontent.com/7459753/45515749-e6fd4680-b75d-11e8-87d4-09cb32cb809b.jpg)
![w_sup_mark](https://user-images.githubusercontent.com/7459753/45515750-e6fd4680-b75d-11e8-9026-2f90140cc404.jpg)

**Comment by miguelsousa** [comment]
Fixed in version 2.040.
